packages/jest-runner/src/jest-test-runner.ts

```diff
@@ -106,7 +106,7 @@ export class JestTestRunner implements TestRunner {
     }
     try {
       const { dryRunResult, jestResult } = await this.run({
-        jestConfig: withCoverageAnalysis(this.jestConfig, coverageAnalysis),
+        jestConfig: withCoverageAnalysis({ ...this.jestConfig, bail: true }, coverageAnalysis),
```

Comment:
It might make sense for `disableBail` to also be added to dry run options and used here. That is what I would expect if I were using `disableBail`. That way you can see all the error messages when the dry run fails.

Reply:
Makes sense. It's Done